jdi-light-angular/src/main/java/elements/common/BasicSpinner.java

```diff
@@ -0,0 +1,6 @@
+package elements.common;
+
+import com.epam.jdi.light.elements.common.UIElement;
+
+public class BasicSpinner extends UIElement {
```

Comment:
make abstract

Reply:
Renamed to ProgressSpinner, do you think that it still needs to be abstract? If so, why?